src/orquestra/integrations/cirq/simulator/_base.py

```diff
@@ -30,6 +30,10 @@ def _prepare_measurable_cirq_circuit(circuit, noise_model):
     Returns:
         circuit to run on a cirq or qsim simulator
     """
+
+    for i in range(circuit.n_qubits):
```

Comment:
Added identity gates to ensure that all the inactive gates are accounted. It was failing otherwise.